### DISCUSSION TITLE
Release 0.10.0

### DISCUSSION THREAD

**Discussion by shumonsharif** [quarkiverse/quarkus-cxf#294] [General]
@dufoli @famod Just wanted to get a conversation started on Release 0.10, based on @famod 's request here:  https://github.com/quarkiverse/quarkus-cxf/pull/293#issuecomment-939897728

@dufoli I don't believe we've introduced any major changes recently code wise, but just wanted to confirm there are no open issues you are working on?  There are some documentation changes I wanted to add, but those are low hanging fruit that can be pushed out.  Let us know please, if you have any reservations about dropping a new release!

**Comment by famod** [comment]
Thanks for taking care of this @shumonsharif!

**Comment by shumonsharif** [comment]
@famod I don't see much [activity](https://github.com/dufoli?tab=overview&from=2021-09-01&to=2021-09-30) from @dufoli since approx mid last month.  I'm thinking of allowing some time to hear from him, are you under any specific timeline?

**Comment by famod** [comment]
@shumonsharif yes I am, going live in two weeks but don't worry, I'll try overriding via `dependencyManagement`.

**Reply by shumonsharif** [reply]
OK, in that case, I will release within the next couple of hours ... will keep you posted!

**Comment by shumonsharif** [comment]
@gastaldi cc: @famod Need your advise, I am trying to create release 0.10.  It appears that the trailing zero got removed during the release process, and instead 0.1 is what got published.  Is a micro version necessary in this case?  Any way to force a "0.10" release or do we need to go with "0.10.0"?

**Reply by gastaldi** [reply]
Probably the radcortez/project-metadata-action is extracting that information as a number, hence why it trims the last 0. Try adding `"` to the version in the https://github.com/quarkiverse/quarkus-cxf/blob/master/.github/project.yml file just in case: 

```yaml
name: Quarkus CXF
release:
  current-version: "0.10"
  next-version: "0.11.0-SNAPSHOT"
```

**Reply by gastaldi** [reply]
But I'd recommend adopting a `Major.Minor.Patch` format anyway 😉

**Reply by shumonsharif** [reply]
Thank you, I'll go with the `Major.Minor.Patch` format - I would agree it works better.

**Reply by gastaldi** [reply]
Sure, using quotes in the version is also a valid option in this case too :)

**Reply by shumonsharif** [reply]
Thanks @gastaldi  I'm fairly used to semantic versioning myself.  :)  I don't recall why we dropped the patch version to begin with, but hopefully @dufoli is good with reintroducing it!  BTW, is it possible to clean up the 0.1 release and remove the corresponding artifacts, both from the project's releases page as well as maven central?

**Reply by gastaldi** [reply]
Unfortunately no, once artifacts hit Maven central, you can't delete them. But I don't think it should be a problem, since the latest is already 0.10.0?

**Reply by shumonsharif** [reply]
Yes, latest is 0.10.0, so we are all set.  Thanks for your input and suggestions!

**Comment by shumonsharif** [comment]
@famod Release 0.10.0 is out.  Hope you are able to leverage it for your go-live!

**Reply by famod** [reply]
Thanks a lot! Jenkins is already building my update PR. 😄
